src/plugins/stake-plugins/thorchainSavers/tcSaversPlugin.ts

```diff
@@ -201,7 +198,42 @@ let thornodeServers: string[] = THORNODE_SERVERS_DEFAULT
 let inboundAddressesLastUpdate: number = 0
 
 export const makeTcSaversPlugin = async (opts: EdgeGuiPluginOptions): Promise<StakePlugin> => {
-  asInitOptions(opts.initOptions)
+  const { ninerealmsClientId } = asInitOptions(opts.initOptions)
+
+  if (!policiesInitialized && !initializingPolicies) {
+    try {
+      initializingPolicies = true
+      const poolsResponse = await fetchWaterfall(thornodeServers, `thorchain/pools`, { headers: { 'x-client-id': ninerealmsClientId } })
+
+      if (!poolsResponse.ok) {
+        const responseText = await poolsResponse.text()
+        throw new Error(`Thorchain could not fetch thornode pools: ${responseText}`)
+      }
+      const poolsJson = await poolsResponse.json()
+      const pools = asThorNodePools(poolsJson)
+      pools.forEach(pool => {
+        if (gt(pool.savers_depth, '0')) {
+          const edgeAsset = tcAssetToEdge(pool.asset)
+          if (edgeAsset == null) return
+          const { pluginId, currencyCode } = edgeAsset
+          const lowerCc = currencyCode.toLowerCase()
+
+          policies.push({
+            ...policyDefault,
+            stakePolicyId: `tcsavers/${pluginId}:${lowerCc}=${pluginId}:${lowerCc}`,
+            rewardAssets: [{ pluginId, currencyCode }],
+            stakeAssets: [{ pluginId, currencyCode }]
+          })
+        }
+      })
+      policiesInitialized = true
+    } catch (e) {
+      console.log('Error initializing Thorchain Savers policies', String(e))
```

Comment:
Would this frequently happen? If not, seems useful to promote this to a console warn or error

Reply:
console.warn is annoying in that it shows as a yellow box in the debugger. It's not critical as it can happen if there is a networking failure